content/en/docs/started/installing-kubeflow.md

```diff
@@ -9,140 +9,249 @@ Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it prov
+        <td>
+          OpenShift
+        </td>
+        <td>
+          <u><a href="https://opendatahub.io/docs/kubeflow.html">Website</a></u>
+          <br>
+          <sub><u><a href="/docs/distributions/openshift/">Legacy Docs</a></u></sub>
+        </td>
+        <td>
+          2022-10-26
+          <br>
+          <sup>[<a href="https://github.com/opendatahub-io/odh-manifests/releases">Release History</a>]</sup>
+        </td>
       </tr>
       <tr>
         <td>Arrikto Kubeflow as a Service</td>
-        <td>Arrikto</td>
-        <td>Fully Managed</td>
-        <td>1.4</td>
-        <td>N/A</td>
-        <td><a href="http://kubeflow.arrikto.com/">External Website</a></td>
+        <td rowspan="2">Arrikto</td>
+        <td>
+          N/A <sup>(fully managed)</sup>
+        </td>
+        <td>
+          <u><a href="https://www.arrikto.com/kubeflow-as-a-service/">Website</a></u>
+        </td>
+        <td>
+          Unknown
+        </td>
       </tr>
       <tr>
         <td>Arrikto Enterprise Kubeflow</td>
-        <td>Arrikto</td>
-        <td>EKS,
-            AKS,
-            GKE
+        <td>
+          ◦ Amazon Elastic Kubernetes Service (EKS)
+          <br>
+          ◦ Azure Kubernetes Service (AKS)
+          <br>
+          ◦ Google Kubernetes Engine (GKE)
         </td>
-        <td>1.4</td>
         <td>
-          <a href="/docs/distributions/ekf/">Docs</a>
+          <u><a href="https://www.arrikto.com/enterprise-kubeflow/">Website</a></u>
         </td>
         <td>
-          <a href="https://www.arrikto.com/enterprise-kubeflow/">External Website</a>
+          2022-10-24
+          <br>
+          <sup>[<a href="https://docs.arrikto.com/Changelog.html">Changelog</a>]</sup>
         </td>
       </tr>
       <tr>
         <td>Charmed Kubeflow</td>
         <td>Canonical</td>
-        <td>Conformant Kubernetes</td>
-        <td>1.6</td>
-        <td><a href="/docs/distributions/charmed/">Docs</a></td>
-        <td><a href="https://charmed-kubeflow.io/">External Website</a></td>
+        <td>
+          All Conformant Kubernetes
+        </td>
+        <td>
+          <u><a href="https://charmed-kubeflow.io/">Website</a></u>
+        </td>
+        <td>
+          2022-09-08
+          <br>
+          <sup>[<a href="https://ubuntu.com/blog/charmed-kubeflow-1-6-available">Blog Post</a>]</sup>
+        </td>
       </tr>
       <tr>
-        <td>Kubeflow on Oracle Container Engine for Kubernetes</td>
+        <td>Kubeflow on Oracle Cloud Infrastructure</td>
         <td>Oracle</td>
-        <td>Oracle Cloud Infrastructure (OCI)</td>
-        <td>1.6</td>
-        <td>N/A</td>
-        <td><a href="https://github.com/oracle-devrel/kubeflow-oke">External Website</a></td>
+        <td>
+          Oracle Container Engine for Kubernetes (OKE)
+        </td>
+        <td>
+          <u><a href="https://github.com/oracle-devrel/kubeflow-oke">Website</a></u>
+        </td>
+        <td>
+          Unknown
```

Comment:
@julioo is there something we can refer here to add the date of your latest release?

Reply:
@julioo just to clarify, we need a publicly listed blog/changelog/release-page that confirms the release date so we can link it.

Reply:
@annajung @thesuperzapper, you're referring to the Oracle Kubernetes Engine release date?
https://docs.oracle.com/en-us/iaas/releasenotes/services/conteng/
for example
https://docs.oracle.com/en-us/iaas/releasenotes/changes/2aca54a0-5594-48f3-8c51-c126d22d4eda/